client/js/views/GameView.vue

```diff
@@ -1191,6 +1191,10 @@ export default {
   grid-area: decks;
 }
 
+.resolving-seven-size {
+    width: 9.5rem;
+}
```

Comment:
This looks good--let's move it down inside the block for `.reveal-top-two {}` block and add a & in front of the selector like 
```suggestion
& .resolving-seven-card {
    width: 9.5rem;
}
```
The & is an alias for the nesting, so that means it'd apply to something with the `.resolving-seven-card` class that's inside the `.reveal-top-two` element.

While we are at it, let's rename the class to resolving-seven-card instead of -size so that we can use the same selector in case we need to apply other styles to these cards besides size